examples/sensorplugin/sampleplugin.cpp

```diff
@@ -40,5 +40,9 @@ void SamplePlugin::Register(class Loader&)
 // dependency here. We can rely on it to provide further dependencies to
 // any plugins it may need.
 QStringList SamplePlugin::Dependencies() {
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
```

Comment:
This change affects Qt starting from 5.14, please fix.

Reply:
Updated 27 occurrences to use Qt::SkipEmptyParts in 5.15. (included) and newer version, the 5.14 and older are using QString::SkipEmptyParts.